src/main/java/jalau/at18/azul/view/TilesSend.java

```diff
@@ -0,0 +1,21 @@
+package jalau.at18.azul.view;
+
+import jalau.at18.azul.BagSender;
+import javax.swing.DefaultListModel;
+import javax.swing.JList;
+import javax.swing.JPanel;
+
+public class TilesSend extends JPanel {
+    private JList<String> countryList;
```

Comment:
JList name " countryList" does not make sense, please review and change according to what it represents

Reply:
already change the name of the list to a proper name